### ISSUE TITLE
Question: Trigger updateFilters with jquery?

### ISSUE THREAD

**Issue by spaszwerg** [mezzoblue/PaintbrushJS#8]
Hi Dave,

i´m a designer from germany and currently working on a litte t-shirt-configuration tool.
With your great library it should be possible.
I´m really not a programmer but can handle some easy jquery scripts.
I use jQuery in this tool as well.
You can find the draft version here: http://new.independentracing.de/

If you click on a colorspot below the shirt, the color of the shirt part should change. Basically this is working with the jQuery script.
The bg-color of the spot is saved into a varriable and this color is injected into the "data-pb-tint-colour"-Attribute. 
But nothing changes! I guess I need to update the filters somehow.
I was going through your demoscripts severeal hours but didn´t manage to get those filters updated.
Basically it would be enough if the filters would be updated on a pagewide mouseup-event.

Maybe you could give me a hint how this can happen.

Sorry, if my request is totally wrong located here. 

Regards, Chris

**Comment by mezzoblue** [comment]
Sorry, can't do individual support requests for code that I haven't touched in 3+ years. 

Take a look at how I'm re-applying effects on the demo page (http://mezzoblue.github.io/PaintbrushJS/demo/) -- particularly this file: http://mezzoblue.github.io/PaintbrushJS/demo/script/playground.js

Hopefully that's enough of a starting point for you.

**Comment by spaszwerg** [comment]
Thanks for your quick reply.
That file is the key to my solution, I guess.
Will go through it some more times.
